StreetHawk/Classes/Location/Internal/SHLocationManager.m

```diff
@@ -172,6 +172,12 @@ + (BOOL)locationServiceEnabledForApp:(BOOL)allowNotDetermined
             //clear location denied flag
             [[NSUserDefaults standardUserDefaults] setObject:@"" forKey:LOCATION_DENIED_SENT];
             [[NSUserDefaults standardUserDefaults] synchronize];
+            // only when sh location module is included in the app and user enable the system location settings, set sh_location_denied as false
+            if([[[NSUserDefaults standardUserDefaults] stringForKey:@"sh_module_location"] compare:@"true"] == NSOrderedSame)
+            {
+                [StreetHawk tagString:@"false" forKey:@"sh_location_denied"];
+                SHLog(@"sh_location_denied set as false due to Location service enabled with sh location module embeded");
```

Comment:
This log is not correct. It's not only include Location module, but also customer enables the permission.